runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -623,6 +632,14 @@ else if (lastTeleportClicked == TeleportWidget.MINIGAME_TELEPORT)
 		lastAnimation = client.getLocalPlayer().getAnimation();
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (config.showVengeanceActive() && event.getGroupId() == WidgetID.ENTERING_HOUSE_GROUP_ID)
```

Comment:
This should actually be for leaving a house, though we don't have a great way to test for that. You could maybe check for being in a house after the scene loading then detect leaving that way.

Reply:
Vengeance is removed upon entering a house; this widget is displayed upon entering a house. Why would you need to check leaving the house?

Reply:
Oh I read the wiki wrong. This is fine.